src/app/page.tsx

```diff
@@ -1,25 +1,18 @@
-// import Link from "next/link"
 import React from "react"
-// import NavBar from "../components/NavBar"
 import Header from "@/components/Header"
+import About from "@/components/About"
+import Banner from "@/components/Banner"
```

Comment:
Added about and banner components just to make it cleaner to read. We aren't "reusing" these components but it makes the landing page code far more readable.